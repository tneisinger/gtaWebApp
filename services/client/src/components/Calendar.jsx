import React from 'react';

const Calendar = () => (
  <div>
    <h1>Calendar</h1>
    <hr/><br/>
    <p>Calendar goes here.</p>
  </div>
);

export default Calendar;
