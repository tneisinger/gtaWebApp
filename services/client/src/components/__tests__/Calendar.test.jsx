import React from 'react';
import { shallow } from 'enzyme';
import renderer from 'react-test-renderer';

import Calendar from '../Calendar';

test('Calendar renders properly', () => {
  const wrapper = shallow(<Calendar/>);
  const element = wrapper.find('p');
  expect(element.length).toBe(1);
  expect(element.text()).toBe('Calendar goes here.');
});

test('Calendar renders a snapshot properly', () => {
  const tree = renderer.create(<Calendar/>).toJSON();
  expect(tree).toMatchSnapshot();
});
