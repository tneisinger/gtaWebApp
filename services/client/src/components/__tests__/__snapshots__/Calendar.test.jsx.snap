// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Calendar renders a snapshot properly 1`] = `
<div>
  <h1>
    Calendar
  </h1>
  <hr />
  <br />
  <p>
    Calendar goes here.
  </p>
</div>
`;
